Kitodo/src/main/webapp/WEB-INF/resources/js/scroll.js

```diff
@@ -157,7 +157,7 @@ function initializeStructureTreeScrolling() {
 }
 
 function scrollToPreviewThumbnail(thumbnail, scrollable) {
-    let thumbnailHeight = thumbnail.parent().parent().height();
+    let thumbnailHeight = thumbnail.closest(".thumbnail-parent").height();
```

Comment:
```suggestion
    let thumbnailHeight = thumbnail.closest(".thumbnail-parent").outerHeight(true);
```
You could use here `outerHeight` to consider the elements margin and change the following rule in kitodo.css:3167.
```
#imagePreviewForm\:thumbnailWrapper .thumbnail-parent {
    display: block;
    margin-bottom: var(--group-margin);
    margin-left: auto;
    margin-right: auto;
}
```
The `scrolltoPreviewThumbnail` method did not take the spacing between the elements into account when computing the scrollTop value. These changes should do the trick.